<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>esptool-gui</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="settingsBox">
      <property name="title">
       <string>Settings</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="1">
        <widget class="QComboBox" name="baudBox">
         <item>
          <property name="text">
           <string>300</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4800</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>19200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>38400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>57600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>74880</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>115200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>230400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>250000</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="portBox"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="fileLabel">
         <property name="text">
          <string>Firmware:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="browseBtn">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="reloadBtn">
         <property name="text">
          <string>Reload</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="portLabel">
         <property name="text">
          <string>Port:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="baudrateLabel">
         <property name="text">
          <string>Baudrate:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="filePathText"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="testFileLabel">
         <property name="text">
          <string>Test Firmware:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="testPathText"/>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="browseTestBtn">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
      <zorder>portLabel</zorder>
      <zorder>portBox</zorder>
      <zorder>reloadBtn</zorder>
      <zorder>baudrateLabel</zorder>
      <zorder>baudBox</zorder>
      <zorder>fileLabel</zorder>
      <zorder>browseBtn</zorder>
      <zorder>filePathText</zorder>
      <zorder>testFileLabel</zorder>
      <zorder>testPathText</zorder>
      <zorder>browseTestBtn</zorder>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="testBtn">
        <property name="text">
         <string>Test</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="uploadBtn">
        <property name="text">
         <string>Upload</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Output</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPlainTextEdit" name="outputText"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="openSerialBtn">
           <property name="text">
            <string>Open Serial</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="closeSerialBtn">
           <property name="text">
            <string>Close Serial</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clearTextBtn">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>403</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>clearTextBtn</sender>
   <signal>clicked()</signal>
   <receiver>outputText</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>377</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
